Helper/Api.php

```diff
@@ -223,12 +223,6 @@ public function sendRequest($request)
 
         $request->setHeaders($headers);
 
-        $this->bugsnag->registerCallback(function ($report) use ($request) {
-            $report->setMetaData([
-                'BOLT API REQUEST' => $request->getData()
```

Comment:
Let's add bolt trace id instead - it would be enough to find all necessary data in the bolt side